<launch>

  <node pkg="chinocon" name="color_image" type="pub_gazo_raspi.py" output="screen">
  </node>

</launch>
